lib/process-services-cloud/src/lib/task/task-header/components/task-header-cloud.component.ts

```diff
@@ -0,0 +1,276 @@
+                    value: this.taskDetails.priority,
+                    key: 'priority',
+                    editable: true
+                }
+            ),
+            new CardViewDateItemModel(
+                {
+                    label: 'ADF_CLOUD_TASK_HEADER.PROPERTIES.DUE_DATE',
+                    value: this.taskDetails.dueDate,
+                    key: 'dueDate',
+                    default: this.translationService.instant('ADF_CLOUD_TASK_HEADER.PROPERTIES.DUE_DATE_DEFAULT'),
+                    editable: true
+                }
+            ),
+            new CardViewTextItemModel(
+                {
+                    label: 'ADF_CLOUD_TASK_HEADER.PROPERTIES.CATEGORY',
+                    value: this.taskDetails.category,
+                    key: 'category',
+                    default: this.translationService.instant('ADF_CLOUD_TASK_HEADER.PROPERTIES.CATEGORY_DEFAULT')
+                }
+            ),
+            new CardViewDateItemModel(
+                {
+                    label: 'ADF_CLOUD_TASK_HEADER.PROPERTIES.CREATED',
+                    value: this.taskDetails.createdDate,
+                    key: 'created'
+                }
+            ),
+            new CardViewMapItemModel(
+                {
+                    label: 'ADF_CLOUD_TASK_HEADER.PROPERTIES.PARENT_NAME',
+                    value: parentInfoMap,
+                    key: 'parentName',
+                    default: this.translationService.instant('ADF_CLOUD_TASK_HEADER.PROPERTIES.PARENT_NAME_DEFAULT'),
+                    clickable: true
+                }
+            ),
+            new CardViewTextItemModel(
+                {
+                    label: 'ADF_CLOUD_TASK_HEADER.PROPERTIES.PARENT_TASK_ID',
+                    value: this.taskDetails.parentTaskId,
+                    key: 'parentTaskId'
+                }
+            ),
+            new CardViewDateItemModel(
+                {
+                    label: 'ADF_CLOUD_TASK_HEADER.PROPERTIES.END_DATE',
+                    value: this.taskDetails.claimedDate,
+                    key: 'endDate'
+                }
+            ),
+            new CardViewTextItemModel(
+                {
+                    label: 'ADF_CLOUD_TASK_HEADER.PROPERTIES.ID',
+                    value: this.taskDetails.id,
+                    key: 'id'
+                }
+            ),
+            new CardViewTextItemModel(
+                {
+                    label: 'ADF_CLOUD_TASK_HEADER.PROPERTIES.DESCRIPTION',
+                    value: this.taskDetails.description,
+                    key: 'description',
+                    default: this.translationService.instant('ADF_CLOUD_TASK_HEADER.PROPERTIES.DESCRIPTION_DEFAULT'),
+                    multiline: true,
+                    editable: true
+                }
+            ),
+            new CardViewTextItemModel(
+                {
+                    label: 'ADF_CLOUD_TASK_HEADER.PROPERTIES.FORM_NAME',
```

Comment:
if we don't know where to fetch this property is better to remove this field for now